src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -0,0 +1,46 @@
+package com.ericsson.eiffel.remrem.generate.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ericsson.eiffel.remrem.protocol.MsgService;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+
+@RestController @RequestMapping("/*") 
+public class RemremGenerateController {
+	
+    @Autowired
+	private List<MsgService> msgServices;
+	private JsonParser parser = new JsonParser();
+	
+	@RequestMapping(value="/{mp}", method = RequestMethod.POST)
+	public JsonElement generate(@PathVariable String mp,@RequestParam("msgType") String msgType,
+           @RequestBody JsonObject bodyJson) {
+		MsgService msgService = getMessageService(mp);
```

Comment:
You have a mix of tabs and spaces in this file. Please only use spaces to indent.